packages/components/src/tree-select/README.md

```diff
@@ -50,51 +51,165 @@ const MyTreeSelect = () => {
 	);
 }
 ```
-
 ## Props
 
-The set of props accepted by the component will be specified below.
-Props not included in this set will be applied to the SelectControl component being used.
```

Comment:
I think this is useful information that we removed by auto-generating the README. Can we make sure it gets included?

Reply:
Mmm right, I think this is something we need to address at the package level. The majority of our components that do pass down props, we don't say that explicitly (e.g. in the canonical Storybook props docs), even though the TS types include them.

Extracted to #67979

Reply:
Sounds good to be addressed as a follow-up 👍